src/lib.rs

```diff
@@ -144,6 +143,21 @@ pub struct DifficultySection {
     pub slider_tick_rate: f32,
 }
 
+impl Default for DifficultySection {
+    /// Create a [DifficultySection](structs.DifficultySection.html) where
+    /// all fields are set to 5 except `slider_multiplier` and `slider_tickrate`,
+    /// as [is the case in the editor](https://github.com/ppy/osu/blame/f517f98ae7bb6e3e60a6ed552a89474b9470344e/osu.Game/Beatmaps/BeatmapDifficulty.cs#L13).
+    fn default() -> Self {
```

Comment:
Looks like this is missing the actual struct creation. Fix this and we're good to go!

Reply:
Making a silly mistake like this is not foreign to me, I'm afraid. I will get to this later.